### PR TITLE
added check for maximumFractionDigits=0

### LINKED ISSUE #379: Round off a decimal to integer format
PR #415

Ref: https://github.com/jquery/globalize/issues/190#issuecomment-69546329

Whole numbers are not rounded.

### PR DESCRIPTION
**Pull request by arvind-kalyan**
Fixes #379

### CODE REVIEW

**Comment by jzaefferer** [comment]
Travis is reporting test failures - can you look into those?

**Comment by arvind-kalyan** [comment]
Okay.

File: src/number/format/integer-fraction-digits.js [hunk 1]
```diff
@@ -43,6 +43,7 @@ return function( number, minimumIntegerDigits, minimumFractionDigits, maximumFra
 			number = number.join( "." );
 		}
 	} else {
+		number = round( number, "round" );
```
Comment (rxaviers) [inline src/number/format/integer-fraction-digits.js]:
```diff
- number = round( number, "round" );
+ number = round( number );
```

File: src/number/format/integer-fraction-digits.js [hunk 2]
```diff
@@ -43,6 +43,7 @@ return function( number, minimumIntegerDigits, minimumFractionDigits, maximumFra
...
 		number = numberTruncate( number );
```
Comment (rxaviers) [inline src/number/format/integer-fraction-digits.js]: We should remove this line and remove the dependency for `numberTruncate`.

**Comment by rxaviers** [comment]
Hi @arvind-kalyan, I have also left a couple of comments above. Just let us know when you have updates. So, we can do another review round.

**Comment by arvind-kalyan** [comment]
Hi, made the changes. Build is failing again. I'll check the code again.

**Comment by rxaviers** [comment]
Two test cases need to have the expected values updated. Feel free to do
it. Thanks

>

-- 
+55 (16) 98138-1582, +1 (415) 568-5854, skype: rxaviers
http://rafael.xavier.blog.br

**Comment by arvind-kalyan** [comment]
I tried to push changes from another location. I'll fix this in 4-5 hours when I reach home.

**Comment by jzaefferer** [comment]
Looks like everything is fine, despite the failures: The CLA check failed since you pushed with different git author information, but once squashed that will disappear, and the other commits are valid. The Travis build is failing due to invalid commit messages - again, squashing commits into one and updating the commit message can address that. I'll leave it to @rxaviers to do that - I may have missed something.

**Comment by arvind-kalyan** [comment]
Hi, I have made the changes. 
@rxaviers  Kindly check.
@jzaefferer What is the correct format for commit messages?

**Comment by jzaefferer** [comment]
That's documented here: http://contribute.jquery.org/commits-and-pull-requests/#commit-guidelines

File: src/number/format/integer-fraction-digits.js [hunk 3]
```diff
@@ -43,7 +43,7 @@ return function( number, minimumIntegerDigits, minimumFractionDigits, maximumFra
 			number = number.join( "." );
 		}
 	} else {
-		number = numberTruncate( number );
+		number = round( number );
```
Comment (rxaviers) [inline src/number/format/integer-fraction-digits.js]:
```diff
define([
-	"../../util/number/truncate",
	"../../util/string/pad"
- ], function( numberTruncate, stringPad ) {
+ ], function( stringPad ) {
```

File: test/unit/number/format/integer-fraction-digits.js [hunk 4]
```diff
@@ -73,4 +73,8 @@ QUnit.test( "should handle inaccurate floating point arithmetics", function( ass
 	assert.equal( formatIntegerFractionDigits( 0.00015, 1, 1, 4, round, null ), "0.0002" );
 });
 
+QUnit.test( "should round off decimal to integer format", function( assert ) {
+	assert.equal( formatIntegerFractionDigits( 100.7, 1, 0, 0, round, null ), "101" );
+});
```
Comment (rxaviers) [inline test/unit/number/format/integer-fraction-digits.js]: Please, move this assertion into the existing "should allow rounding" block.

**Comment by rxaviers** [comment]
:+1:

**Pull request closed without merging**